tests/integration-tests/remote_command_executor.py

```diff
@@ -233,7 +233,7 @@ def get_remote_files(self, *args, **kwargs):
     def clear_log_file(self, path: str):
         """Clear a log file in a specific path."""
 
-        self.run_remote_command(f"sudo truncate -s 0 {path}")
+        self.run_remote_command(f"sudo -u {self.get_user_to_operate_on_file(path)} truncate -s 0 {path}")
```

Comment:
Why do we need this change now? Is it a security improvement or what else?

Reply:
According to the commit description:
> In many of the operating systems ParallelCluster supported, `root` user could bypass any files permissions. However, with RHEL9 and Rocky9, `root` user could not bypass files permissions by default. Therefore, this commit improves integration tests to operate on files with the owner users of the files